client/components/Profile/MyProfile.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { ReactElement, useState } from "react";
+
+import Paper from "@material-ui/core/Paper";
+
+import Button from "@material-ui/core/Button";
+
+import API from "../../util/api";
+import useApi from "../../hooks/useApi";
+import useStyles from "./Profile.styles";
+import ShowComments from "./ShowComments";
+import OnClickInput from "./OnClickInput";
+import Password from "./Password";
+import EditableAvatar from "./EditableAvatar";
+
+interface UrlParam {
+  username: string;
+}
+interface Comment {
+  movieName: string;
+  date: number;
+  name: string;
+  body: string;
+  stars: number;
+  _id: number;
+}
+
+interface User {
+  username: string;
+  firstName: string;
+  email: string;
+  lastName: string;
+  picture: string;
+}
+
+const MyProfile = (): ReactElement => {
+  const { resData: data } = useApi<User, void>(`/user/`, { hotReload: true });
+  const { username } = data || {};
+  const classes = useStyles({});
+  const [changingPassword, setChangingPassword] = useState(false);
+
+  const updateInfo = (value: string, name: string): void => {
+    if (value && name) {
+      API.put("/edit-profile", { [name]: value })
+        // .then(() => {
+        //   console.log("");
+        // })
+        .catch((e) => {
+          console.error(e);
+        });
+    }
+  };
+  return (
```

Comment:
space to breath a bit
```suggestion
  };
  
  return (
```